cmd/create/oidcprovider/cmd.go

```diff
@@ -148,11 +148,19 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	mode := args.mode
+	autoMode := cmd.Flags().Changed("mode") && mode == "auto"
+	forceManual := false
+
 	// Check to see if IAM operator roles have already created
 	missingRoles, err := validateOperatorRoles(awsClient, cluster)
 	if err != nil {
-		reporter.Errorf("Failed to validate operator roles exist: %s", err)
-		os.Exit(1)
+		reporter.Errorf("Failed to validate if operator roles exist: %s", err)
```

Comment:
Can we change this message to just print for other errors and for access denied we can print saying that "access is denied" trying running manually .. something like that?